master/master.cfg

```diff
@@ -1393,21 +1352,48 @@ def prioritize_builders(buildmaster, builders):
             return 4
         return 5
 
-    builders.sort(key=importance)
+    return list(sorted(builders, key=importance))
 
-    print("prioritize_builders:")
-    for b in builders:
-        print("  PB: %s (%s) -> pri %d" %
-              (b.name, b.config.builder_type.builder_label(), importance(b)))
 
-    return builders
+c['prioritizeBuilders'] = prioritize_builders
 
 
-c['prioritizeBuilders'] = prioritize_builders
+# GitHub pull request filter
+
+class SafeGitHubEventHandler(GitHubEventHandler):
+    def handle_pull_request(self, payload, event):
+        pr = payload['pull_request']
+        try:
+            # Skip anything with the 'skip_buildbots' label
+            if any(label['name'] == 'skip_buildbots' for label in pr['labels']):
+                return self.skip()
+
+            # Test anything (even external) that has 'halidebuildbots' as a reviewer.
+            if any(r['login'] == 'halidebuildbots' for r in pr['requested_reviewers']):
+                return super().handle_pull_request(payload, event)
+
+            # Skip external PRs
```

Comment:
By 'external' I presume you mean "fork rather than branch"

Reply:
Yes.